src/PhpSpreadsheet/Reader/Gnumeric.php

```diff
@@ -639,27 +691,29 @@ private static function addStyle2(array &$styleArray, string $key1, string $key,
         }
     }
 
-    private static function parseBorderAttributes($borderAttributes)
+    private static function parseBorderAttributes(?SimpleXMLElement $borderAttributes): array
     {
         $styleArray = [];
-        if (isset($borderAttributes['Color'])) {
-            $styleArray['color']['rgb'] = self::parseGnumericColour($borderAttributes['Color']);
-        }
+        if ($borderAttributes !== null) {
+            if (isset($borderAttributes['Color'])) {
+                $styleArray['color']['rgb'] = self::parseGnumericColour($borderAttributes['Color']);
+            }
 
-        self::addStyle($styleArray, 'borderStyle', $borderAttributes['Style']);
+            self::addStyle($styleArray, 'borderStyle', $borderAttributes['Style']);
+        }
 
         return $styleArray;
     }
 
-    private function parseRichText($is)
+    private function parseRichText(string $is): RichText
```

Comment:
I do love the fact that we can use this exercise as an excuse to apply typehinting as well... always try to leave the codebase in a better state than you found it

Reply:
I think that was one of the grandfathered Phpstan exceptions. That baseline file is pretty useful.